geopandas/io/arrow.py

```diff
@@ -638,27 +688,33 @@ def _read_parquet(path, columns=None, storage_options=None, **kwargs):
     )
 
     path = _expand_user(path)
+    schema, metadata = _read_parquet_schema_and_metadata(path, filesystem)
+    geo_metadata = _validate_metadata(metadata)
+
+    bbox_filter = _get_parquet_bbox_filter(geo_metadata, bbox) if bbox else None
```

Comment:
```suggestion
    bbox_filter = _get_parquet_bbox_filter(geo_metadata, bbox) if bbox is not None else None
```

that will allow passing the `bbox` as a numpy array of 4 elements (otherwise the thruthiness of a numpy array raises), and I think it would be nice to support that so one can pass the `gdf.total_bounds` of another dataframe